app/lib/file_parse_service.rb

```diff
@@ -11,6 +11,9 @@ class FileParseService
   # * *returns*
   #   - (Hash) => Status object with http status_code and optional error message
   def self.run_parse_job(study_file, study, user, reparse: false, persist_on_fail: false)
```

Comment:
Blocker: this method declaration needs `obsm_key: nil` added to it so we can call it from `StudyFile#call_ingest_on_updated_clustering_fragments`.